subjects/thirdtimeisacharm/README.md

```diff
@@ -0,0 +1,46 @@
+# ThirdTimeIsACharm
+
+### Instructions
+
+Write a function `ThirdTimeIsACharm()` that takes a string and returns another string with every third character.
+
+- Prints the Output followed by newline `\n`.
+- If the string is empty, return newline `\n`.
+- If there is no third character, return `\n`
```

Comment:
Missing "new line" after "return" so it follows the previous sentence?
Missing a punctuation mark/full stop at the end of the sentence.   
"If there is no third character, return `\n`"." "

Reply:
it's fixed